components/browser_kit.rst

```diff
@@ -351,6 +351,19 @@ dedicated web crawler or scraper such as `Goutte`_::
         '.table-list-header-toggle a:nth-child(1)'
     )->text());
 
+Dealing with HTTP responses
+~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+When using the BrowserKit component, you may need to deal with responses of
+the requests you made. To do so, you simply have to make a request, then call
```

Comment:
```suggestion
the requests you made. To do so, you need to call
```

Reply:
Thank you for the suggestion!
About version 4.4, the `getResponse()` method didn't exist at this time. That's why I targeted 5.4 :slightly_smiling_face:

Reply:
https://github.com/symfony/symfony/blob/4.4/src/Symfony/Component/BrowserKit/Client.php#L234 🤔

Reply:
Ooowh that was moved to `AbstractBrowser` after, I did not check `Client`! Thank you, I'm rebasing!